opening_hours/utils/hauki_exporter.py

```diff
@@ -14,128 +16,110 @@ class HaukiResource:
     id: int | None
     name: HaukiTranslatedField
     description: HaukiTranslatedField
-    address: HaukiTranslatedField | None
-    children: list[int]
     parents: list[int]
     organization: str
     origin_id: str
-    origin_data_source_name: str
-    origin_data_source_id: str
-    resource_type: ResourceType = ResourceType.RESERVABLE
 
-    def convert_to_request_data(self):
+
+class ReservationUnitHaukiExporter:
+    reservation_unit: ReservationUnit
```

Comment:
Based on the fact that this class only needs the `reservation_unit` instance to work, these methods could be included in a `ReservationUnitActions` class.